misk/src/test/kotlin/misk/web/ssl/Http2ConnectivityTest.kt

```diff
@@ -129,6 +139,33 @@ class Http2ConnectivityTest {
     }
   }
 
+  /** Confirm we don't page oncall and we record a 499 in the metrics. **/
+  @Test
+  fun clientTimeoutWritingTheRequest() {
+    val http1Client = client.newBuilder()
+        .protocols(listOf(Protocol.HTTP_1_1))
+        .writeTimeout(1, TimeUnit.MILLISECONDS)
+        .build()
+    val requestBody = object : RequestBody() {
+      override fun contentType() = "text/plain;charset=utf-8".toMediaType()
+
+      override fun writeTo(sink: BufferedSink) {
+        for (i in 0 until 1024 * 1024) {
+          sink.writeUtf8("impossible\n")
```

Comment:
I'm not sure if this is the best way to trigger this scenario. I'm open to other ideas.

Reply:
another option: override contentLength() to be 1, and call `sink.close()` without writing any bytes?

Reply:
I tried that but okhttp wasn't happy with that.